test/functional/p2p_node_network_limited.py

```diff
@@ -9,13 +9,12 @@
     - send a block within 288 + 2 of the tip
     - disconnect peers who request blocks older than that."""
 from test_framework.messages import CInv, MSG_BLOCK, msg_getdata, msg_verack, NODE_NETWORK_LIMITED, NODE_WITNESS
-from test_framework.mininode import P2PInterface, mininode_lock
+from test_framework.mininode import P2PInterface
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
     assert_equal,
     disconnect_nodes,
-    connect_nodes,
-    wait_until,
+    connect_nodes
```

Comment:
Any reason to remove the trailing comma in this line? (Same in the other changes)

Reply:
No reason, I have since added back the trailing commas.